src/MainContent/Product.js

```diff
@@ -29,7 +30,8 @@ function Product(props) {
               <Col xs="4">
                 <Row>
                   <img className="product-popup-image" src={props.imgSrc}></img>
-                </Row>
+                  <div className="product-popup-price">{props.price} €</div>
+                </Row>   
```

Comment:
butu geriau jei perdetum i (linkas) ir pakeistum css kad rodytu apacioje.
![image](https://user-images.githubusercontent.com/57706951/113024070-ff53d600-918e-11eb-9224-05b666b3af77.png)

Reply:
Va kas gavosi kai pakeičiau ir mažinu browserio langą: naujausias commit - 
![image](https://user-images.githubusercontent.com/53144067/113026052-5064c980-9191-11eb-981f-3856b9408589.png)

pirmas commit kaip buvo prieš pakeitimus mažinant langą -
![image](https://user-images.githubusercontent.com/53144067/113026295-94f06500-9191-11eb-9b06-b072c5af3d2b.png)

🤔

Reply:
Velnias dabar iš vis kai pradėjau žiūrėt tai viskas stumdosi, čia ant masterio kai info pridedu daugiau... 
![image](https://user-images.githubusercontent.com/53144067/113029620-4ba21480-9195-11eb-9404-e54d1355b227.png)

Reply:
css madness :DDDDDD

Reply:
jiroj komentara imeciau

Reply:
cia nuotrauka persimeta nes css bando talpint viska vaizde. tos problemos nebus jeidasdjisadiashdosaduiadhosidha idesi tarpus

Reply:
as approvinu nes is tavo puses cia viskas gerai o kazkas dar padarys ta bugo taska jiroj